tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```

Comment:
Add test cases for multiple patients
Add test a test case for patients with a single measurement
Add a test case with values that are closer to the real data (up to 100)

Reply:
will do!